ingress/controller.go

```diff
@@ -45,7 +45,11 @@ func (c *controller) Run() {
 	}
 
 	// initialize with current state of ingress
-	c.updateLoadBalancer()
+	err = c.updateLoadBalancer()
```

Comment:
Ok I'll fix this - you're right there is a race condition on the ingress selection/update.